### PR TITLE
makes marker beacons get turned off by veil

### LINKED ISSUE #13260: veil or spray paint doesn't remove light from marker beacons.
## Test Merges: 
Not Supplied / None
## Reproduction:
Description: Come on, really? Spray painting will darken light sources, but the marker beakers are immune? You're literally covering up any way from them to even use or reflect light. Veil turns glowshrooms into shadowshrooms, it turns off all lights that aren't fire based. Even glowsticks. But a miner with marker beacons can just spam them and be happy. Nah, if bio-lights get turned off so should marker beacons.
https://puu.sh/IIoT3/0b89517234.png
 Submitted by: Rg4 (185114950354731008)

### PR DESCRIPTION
**Pull request by UselessTheremin**
fixes #13260
decided not to do spraypaint since you can also just pick em up to turn em off

:cl:  
bugfix: veil now darkens marker beacons  
/:cl:

### CODE REVIEW

**Review by ToasterBiome — APPROVED** [review]
looks good, seems unintended that they were left out

**Comment by SomeguyManperson** [comment]
That appears to bethe item for the stack, not the structure version that emits light
Also veil's initial checks only kill lights that are effecting the tile the sling is standing on iirc so this would only cut adjacent marker beacons

File: yogstation/code/modules/antagonists/shadowling/shadowling_abilities.dm [hunk 1]
```diff
@@ -175,6 +175,10 @@
 		if(istype(LO, /obj/machinery/power/floodlight))
 			var/obj/machinery/power/floodlight/FL = LO
 			FL.change_setting(2) // Set floodlight to lowest setting
+		if(istype(LO, /obj/item/stack/marker_beacon))
```
Comment (ToasterBiome) [inline yogstation/code/modules/antagonists/shadowling/shadowling_abilities.dm:178]:
```suggestion
		if(istype(LO, /obj/structure/marker_beacon))
```

**Review by ToasterBiome — CHANGES_REQUESTED** [review]

File: yogstation/code/modules/antagonists/shadowling/shadowling_abilities.dm (hunk 1 above)
Comment (ToasterBiome) [reply yogstation/code/modules/antagonists/shadowling/shadowling_abilities.dm:178]: Also maybe turn this into a loop like glowshrooms as theos said if you want it to be bigger radius

**Comment by bruhlookatthisdood** [comment]
so uhhhhh
hows it goooin?

**Comment by JamieD1** [comment]
> so uhhhhh hows it goooin?

Awaiting Author fix

**Comment by ToasterBiome** [comment]
@UselessTheremin

**Comment by ToasterBiome** [comment]
Theremin says they're done with yogs, closing

**Pull request closed without merging**